packages/manager/modules/sign-up/src/form/activity/constants.js

```diff
@@ -0,0 +1,71 @@
+// @source: https://en.wikipedia.org/wiki/VAT_identification_number
+export const COUNTRIES_VAT_LABEL = {
+  AT: 'UID',
+  BE: 'TVA',
+  CY: 'ΦΠΑ',
+  CZ: 'DIČ',
+  DE: 'USt-IdNr.',
+  DK: 'CVR',
+  EE: 'KMKR',
+  GR: 'ΑΦΜ',
+  ES: 'NIF',
+  FI: 'ALV nro',
+  FR: 'TVA',
+  HR: 'PDV-ID',
+  HU: 'ANUM',
+  IE: 'VAT',
+  IT: 'IVA',
+  LT: 'PVM kodas',
+  LU: 'TVA',
+  LV: 'PVN',
+  MT: 'Vat No.',
+  NL: 'Btw-nr.',
+  PL: 'NIP',
+  PT: 'NIF',
+  RO: 'CIF',
+  SE: 'Momsnr.',
+  SI: 'ID za DDV',
+  SK: 'IČ DPH',
+  AL: 'NIPT',
+  MK: 'ЕДБ',
+  AU: 'ABN',
+  BY: 'УНП / UNP',
+  CA: 'BN / NE',
+  IS: 'VSK / VASK',
+  IN: 'VAT TIN / CST TIN',
+  ID: 'NPWP',
+  KZ: 'БСН / BIN',
+  NZ: 'GST/IRD',
+  NG: 'VAT',
+  NO: 'Orgnr',
+  PH: 'TIN',
+  RU: 'ИНН',
+  SM: 'C.O.E.',
+  RS: 'PIB',
+  CH: 'MWST / TVA / IVA',
+  TR: 'KDV',
+  UA: 'ІНПП',
+  GB: 'VAT Reg No',
+  UZ: 'СТИР',
+  AR: 'CUIT',
+  BO: 'NIT',
+  BR: 'CNPJ',
+  CL: 'RUT',
+  CO: 'NIT',
+  EC: 'RUC',
+  SV: 'NIT',
+  GT: 'NIT',
+  HN: 'RTN',
+  MX: 'RFC',
+  NI: 'RUC',
+  PA: 'RUC',
+  PY: 'RUC',
+  PE: 'RUC',
+  DO: 'RNC',
+  UY: 'RUT',
+  VE: 'RIF',
```

Comment:
No library or API that already has this kind of knowledge I guess

Reply:
Indeed, I don't find a library that handle it.
And for API, they have such informations but it's not exposed and don't plan to expose it